### PR TITLE
Add PR template on project

### PR DESCRIPTION
**Pull request by kazuhirodk**
## Description
Add template for PR

## Issue
https://github.com/sagar-jadhav/go-examples/issues/30

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+<!-- Please describe the tests that you ran to verify your changes.
+Provide instructions so we can reproduce.
+Please also list any relevant details for your test configuration -->
```
Comment (kazuhirodk) [inline pull_request_template.md]:
```suggestion
Please also list any relevant details for your test configuration
```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- [ ] Test A
+- [ ] Test B
```
Comment (kazuhirodk) [inline pull_request_template.md]:
```suggestion
- [ ] Test B  -->
```

**Review by sagar-jadhav — APPROVED** [review]
Add link to Pull Request Template in README.

**Review by sagar-jadhav — CHANGES_REQUESTED** [review]
Incorporate Review Comments

**Review by rajkumarGosavi — APPROVED** [review]

**Comment by kazuhirodk** [comment]
I added the template link in the readme, but in the default branch it will already be applied automatically when a pull request is opened 😸

**Review by sagar-jadhav — APPROVED** [review]

**Review by rajkumarGosavi — COMMENTED** [review]
Great!

**Pull request merged**